playbooks/configure-apt-sources.yml

```diff
@@ -126,6 +146,20 @@
       when:
         - apt_artifact_found | bool
         - apt_artifact_enabled | bool
+        - apt_artifact_mode == "strict"
+
+    # Set rpco_apt_sources_restore_source to 'local' to have the
+    # original apt sources file from the deploy host be pushed out
+    # to all hosts and containers.
+    - name: Restore the original sources file (artifact 'loose' mode)
```

Comment:
I don't think this task should be part of this PR. The restore actions we're proposing in #2720 are aiming to address this issue. From the POV of this PR it make sense to simply make the modifications as it is today when enabled, unless we want to merge these two PRs.

Reply:
As above.